.github/workflows/_build_release_pypi.yml

```diff
@@ -117,7 +117,6 @@ jobs:
         with:
           target: ${{ matrix.platform.target }}
           args: ${{ matrix.platform.os == 'linux' && '--zig' || '' }} --release --out dist
-          sccache: "true"
```

Comment:
curious, why was this removed?

Reply:
Nevermind I see. Because we're caching everything else, we don't need `sccache` since it serves as a build cache for Rust, iiuc.